### PR TITLE
Fix OOM error and add basic stress test setup script

### PR DESCRIPTION
**Pull request by JoeCap08055**
# Description
- Bump `@dsnp/graph-sdk` dependency to version `1.1.0` to resolve OOM crash.
- Scripting support for creating a stress test scenario.
    - [x] Basic script for creating provider & large number of MSAs on-chain
    - [x] Script writes JSON response files for mock webhook service
        - Script outputs block # where "famous" user was created
        - Response files include:
            - Many users follow one "famous" user ("connectionTo")
            - Famous user response includes "connectionFrom" all other users

### CODE REVIEW

**Comment by JoeCap08055** [comment]
Note, when I tried this for 7k users on a local instant seal node, it hung around block 1800 with a pending extrinsic stuck on the chain.

File: src/processor/reconnection-graph.service.ts [hunk 1]
```diff
@@ -314,7 +315,12 @@ export class ReconnectionGraphService {
   ): Promise<ConnectAction[]> {
     const dsnpKeys: DsnpKeys = await this.formDsnpKeys(dsnpUserId);
     const actions: ConnectAction[] = [];
-    // this.logger.debug(`Graph connections for user ${dsnpUserId.toString()}: ${JSON.stringify(graphConnections)}`);
+    const delegationResult: Option<Vec<SchemaGrantResponse>> = await this.blockchainService.rpc('msa', 'grantedSchemaIdsByMsaId', dsnpUserId, providerId);
+    if (delegationResult.isNone) {
+      this.logger.log(`User ${dsnpUserId} has no delegations to provider ${providerId}`);
+      return actions;
+    }
+    const delegations = delegationResult.unwrap().toArray();
```
Comment (JoeCap08055) [inline src/processor/reconnection-graph.service.ts:323]: This file contains the only code change to the service. Originally we were querying the RPC to get the current user's delegations inside the closure for each connection; here we've refactored it so we only need to call it once. We still can't avoid the RPC call to get schema-specific delegations for each connection, but in my test case of 7k connections, this reduced RPC calls here from 14k -> 7,001.

File: setup/testing/package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  },
+  "dependencies": {
+    "@dsnp/graph-sdk": "^1.1.0",
+    "@frequency-chain/api-augment": "^1.11.1",
+    "@polkadot/api-base": "^10.12.4",
+    "@polkadot/keyring": "^12.6.2",
+    "@polkadot/types": "^10.12.4",
+    "@polkadot/util": "^12.6.2",
+    "@polkadot/util-crypto": "^12.6.2",
+    "frequency-scenario-template": "file:../../../frequency-scenario-template",
```
Comment (enddynayn) [inline setup/testing/package.json:17]: reminder to update.

**Review by aramikm — APPROVED** [review]
Looks good to me

**Review by saraswatpuneet — APPROVED** [review]

**Pull request merged**